### PR TITLE
Reword some docstrings.

### PR DESCRIPTION
**Pull request by anntzer**
For many parameters, reword their description from "Control the foo" or
"Specifies the foo" to "The foo", which is just as clear.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/sphinxext/plot_directive.py [hunk 1]
```diff
@@ -46,11 +46,11 @@
 The ``plot`` directive supports the following options:
 
     format : {'python', 'doctest'}
-        Specify the format of the input
+        The format of the input.
 
     include-source : bool
         Whether to display the source code. The default can be changed
-        using the `plot_include_source` variable in conf.py
+        using the `plot_include_source` variable in conf.py.
```
Comment (timhoffm) [inline lib/matplotlib/sphinxext/plot_directive.py]:
```suggestion
        using the `plot_include_source` variable in :file:ˋconf.pyˋ.
```

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -3514,7 +3514,7 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
             fliers default to 'b+'  If you want more control use the
             flierprops kwarg.
 
-        vert : bool, optional (True)
+        vert : bool, optional, default: True
             If `True` (default), makes the boxes vertical. If `False`,
```
Comment (story645) [inline lib/matplotlib/axes/_axes.py]: doesn't need the  default in `True`(default) anymore

File: lib/matplotlib/mlab.py [hunk 3]
```diff
@@ -152,12 +152,12 @@ def detrend(x, key=None, axis=None):
...
 
     key : {'default', 'constant', 'mean', 'linear', 'none'} or function
-        Specifies the detrend algorithm to use. 'default' is 'mean', which is
-        the same as `detrend_mean`. 'constant' is the same. 'linear' is
-        the same as `detrend_linear`. 'none' is the same as
-        `detrend_none`. The default is 'mean'. See the corresponding
-        functions for more details regarding the algorithms. Can also be a
-        function that carries out the detrend operation.
+        The detrending algorithm to use. 'default' is 'mean', which is the
+        same as `detrend_mean`. 'constant' is the same. 'linear' is the same
```
Comment (story645) [inline lib/matplotlib/mlab.py]: missing what `constant` is the same as. 
Also, are these equivalent keywords: 
`linear` and `detrend_linear` or is the latter the algorithm specified by the former?

File: lib/matplotlib/mlab.py [hunk 4]
```diff
@@ -629,10 +629,9 @@ def _single_spectrum_helper(x, mode, Fs=None, window=None, pad_to=None,
         version of the segment.
 
     sides : {'default', 'onesided', 'twosided'}
-        Specifies which sides of the spectrum to return.  Default gives the
-        default behavior, which returns one-sided for real data and both
-        for complex data.  'onesided' forces the return of a one-sided
-        spectrum, while 'twosided' forces two-sided.
+        Which sides of the spectrum to return. 'default' is one-sided for real
```
Comment (story645) [inline lib/matplotlib/mlab.py]: is it possible to have bulleted list?
* 'default': one-sided for real data and two-sided for complex data.
* 'onesided' forces the return of a one-sided spectrum
* 'twosided' forces two-sided

File: lib/matplotlib/axes/_axes.py [hunk 5]
```diff
@@ -1258,12 +1258,12 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
...
 
         orientation : {'horizontal', 'vertical'}, optional
-            Controls the direction of the event collections:
+            The direction of the event collections:
 
-                - 'horizontal' : the lines are arranged horizontally in rows,
-                  and are vertical.
-                - 'vertical' : the lines are arranged vertically in columns,
-                  and are horizontal.
+            - 'horizontal': the lines are arranged horizontally in rows,
```
Comment (story645) [inline lib/matplotlib/axes/_axes.py]: find "are vertical and arranged horizontally in rows" to be a little counterintutive

File: lib/matplotlib/axes/_axes.py (hunk 2 above)
Comment (anntzer) [reply lib/matplotlib/axes/_axes.py]: done (and reworded to be consistent with violinplot)

File: lib/matplotlib/mlab.py (hunk 3 above)
Comment (anntzer) [reply lib/matplotlib/mlab.py]: reworded for constant.  passing 'linear' is the same as passing `detrend_linear`.

File: lib/matplotlib/axes/_axes.py (hunk 5 above)
Comment (anntzer) [reply lib/matplotlib/axes/_axes.py]: I didn't change the wording, feel free to propose something better.

File: lib/matplotlib/mlab.py (hunk 4 above)
Comment (anntzer) [reply lib/matplotlib/mlab.py]: That just seems very verbose when the interpretation of 'onesided' and 'twosided' are actually pretty obvious (only 'default' really needs an explanation).

File: lib/matplotlib/axes/_axes.py (hunk 2 above)
Comment (timhoffm) [reply lib/matplotlib/axes/_axes.py]: Holds also for a number of other cases in the PR, but we have to clean that up globally anyway.

File: lib/matplotlib/mlab.py (hunk 4 above)
Comment (story645) [reply lib/matplotlib/mlab.py]: Agree it's verbose, but I think bulleted lists are easier to parse especially here where it becomes a left is the key, right is the meaning list.

File: lib/matplotlib/axes/_axes.py (hunk 5 above)
Comment (story645) [reply lib/matplotlib/axes/_axes.py]: Hmm, the problem is i don't even know what this wording is getting at, so I'll agree to table it for a different PR.

File: lib/matplotlib/mlab.py (hunk 4 above)
Comment (anntzer) [reply lib/matplotlib/mlab.py]: I would actually write the whole thing as
> Whether to return a 'onesided' or 'twosided' spectrum.  The 'default' is a 'onesided' spectrum for real data and 'twosided' for complex data.

"'foo' means foo" just reads horribly to me.

Comment (story645) [reply lib/matplotlib/mlab.py]: But here the idea is 'foo' always forces a return of that type? So 'twosided' returns 'twosided' even on real data?

Comment (anntzer) [reply lib/matplotlib/mlab.py]: Yes?

Comment (story645) [reply lib/matplotlib/mlab.py]: So your preferred wording leaves out that forcing behavior.

Comment (anntzer) [reply lib/matplotlib/mlab.py]: I honestly fail to see the semantic content of the word "forcing" here; what ambiguity would this word resolve?  What other behavior do you think the users could imagine if that word was removed?

Comment (story645) [reply lib/matplotlib/mlab.py]: What I mean by forcing is that the keyword chosen == type returned (and I agree/don't think forcing is even the right word here). My concern is not about behaviors that users can imagine, it's about users potential uncertainty about what this argument does if it's not explicitly spelled out. But also, won't hold up the PR on this so I'll merge if you're done w/ changes.

Comment (anntzer) [reply lib/matplotlib/mlab.py]: I... genuinely don't understand what is the potential uncertainty that you refer to. In any, if you're fine with the changes here I think this PR can go in as is.

**Pull request merged**